runner/host/processes.go

```diff
@@ -9,7 +9,12 @@ import (
 
 var _ runner.Runner = &Process{}
 
-type Process struct{}
+// Process represents a single OS Process
+type Process struct {
```

Comment:
Using the `Runner` type here to store one result from Run() isn't quite it yet. We set these fields as parameters when we create a `Runner`, so when we call the `process.Run()` later it has everything it needs. Put another way, this is the *Process Runner* not a data representation of a process's pid, ppid, and name.

Instead you'll need a data container to store and render the collection of processes that process.Run() generates. Try:

```
type proc struct {
        Name string `json:"name"`
	PID  int    `json:"pid"`
	PPID int    `json:"ppid"`
}
```

Finally, you can return a flat slice of results like `[]proc` - or `map[string]proc` if you decide to use a compound key as mentioned in other comments. There's no need to store anything on the `Process` struct because `Process.Run()` doesn't need any params. The results get returned on an immutable `Op{}` so no need to store those on `Process` either.